xbmc/interfaces/legacy/Control.h

```diff
@@ -2593,7 +2586,8 @@ namespace XBMCAddon
     ///
     ///--------------------------------------------------------------------------
     /// @python_v13 New function added.
-    /// @python_v16 Deprecated **focusTexture** and **noFocusTexture**. Use **focusOnTexture** and **noFocusOnTexture**.
+    /// @python_v18 Deprecated **focusTexture** and **noFocusTexture**. Use **focusOnTexture** and **noFocusOnTexture**.
+    /// @python_v19 Removed **focusTexture** and **noFocusTexture**. Use **focusOnTexture** and **noFocusOnTexture**.
```

Comment:
Removed means it was deprecated before (with alternatives) so the note is redundant here

Reply:
i think you mentioned that before :-)
will fix.